community/labels.yaml

```
@@ -406,11 +401,22 @@ default:
         - name: sig/docs
       prowPlugin: label
       target: both
-    - addedBy: human
+
+    - addedBy: anyone
       color: 0052cc
       description:
-        Issues or PRs related to the Custom Notebook Image ODH feature.
-      name: wg/cnbi
+        Issues or PRs related to the Custom Runtime Environment (fka Custom Notebook Image)
```

```suggestion
        Issues or PRs related to the Custom Runtime Environment
```
The information is already there in `previously` (I'm not sure where the description of labels is shown in gh UI anyway ?)

---

> I'm not sure where the description of labels is shown in gh UI anyway

It is: https://github.com/thoth-station/meteor-operator/labels?q=wg
but the `previously` references aren't.

So, I think it makes sense to leave that comment in the description for the time being